tests/test_files/calculator/calculator.py

```diff
@@ -1,5 +1,47 @@
-# TODO: Implement a calculator program, that can add, subtract, multiply and divide two numbers.
-#  - Numbers and operators given from the user as the command line arguments.
-#  - Consider using argpasre library.
-#  - Consider using eval builtin function.
-#  - Print or log the result.
+"""Calculator runnable
```

Comment:
Yes, this file looks way better. This part of the documentation can be moved to the function that calculates the result.

Reply:
Which part of the documentation are you referring to?

Reply:
Calculate runnable ... to the console